govuk_content_api.rb

```diff
@@ -89,43 +89,11 @@ def set_expiry(duration = DEFAULT_CACHE_TIME, options = {})
 
   get "/local_authorities.json" do
     set_expiry LONG_CACHE_TIME
```

Comment:
Good spot - no, that wasn't intentional :) `LONG_CACHE_TIME` [(1 hr instead of 15 mins)](https://github.com/alphagov/govuk_content_api/blob/190a2cfe1b395d973a6fe33571c34ba067c4d64d/govuk_content_api.rb#L40-L41) seems fine for these still so I'll add it back into the other one.